docs/data/quick-start.md

```diff
@@ -0,0 +1,791 @@
+
+    amplitude.track(
+        BaseEvent(
+            event_type="type of event",
+            user_id="USER_ID",
+            device_id="DEVICE_ID",
+            event_properties={
+                "source": "notification"
+            }
+    ))
+    ```
+
+=== "React Native"
+
+    ```ts title="TypeScript"
+
+    import { track } from '@amplitude/analytics-react-native';
+
+    const eventProperties = {
+        buttonColor: 'primary',
+    };
+    track('Button Clicked', eventProperties);
+    ```
+
+=== "Flutter"
+
+    ```dart title="Dart"
+
+    amplitude.logEvent('BUTTON_CLICKED', {"Hover Time": "100ms"});
+    ```
+
+=== "Unity"
+
+    ```c# title="C#"
+    import 'package:amplitude_flutter/amplitude.dart';
+
+    amplitude.logEvent('MyApp startup', eventProperties: {
+      'friend_num': 10,
+      'is_heavy_user': true
+    });
+    ```
+
+=== "Unreal"
+
+    ```c++ title="C++"
+
+    TArray<FAnalyticsEventAttribute> AppendedAttributes;
+    AppendedAttributes.Emplace(TEXT("Test Event Prop key1"), TEXT("Test Event value1"));
+    AppendedAttributes.Emplace(TEXT("Test Event Prop key2"), TEXT("Test Event value2"));
+    FAnalytics::Get().GetDefaultConfiguredProvider()->RecordEvent(TEXT("Game Started"), AppendedAttributes);
+    ```
+
+=== "GO"
+
+    ```go title="GO"
+
+    amplitude.Track(amplitude.Event{
+           EventType: "Button Clicked",
+           EventOptions: amplitude.EventOptions{
+           UserID:   "user-id",
+           DeviceID: "device-id",
+        },
+        EventProperties: map[string]interface{}{"source": "notification"},
+    })
+
+    ```
+
+## 3. Check for success
+#### Ingestion Debugger
+    1. Login to you Amplitude Analytics Orgnization.
+    2. Click the `Data Source tab` at the bottom of the navigation bar
+    3. Choose the `Ingestion Debugger` section. You can check the Requests or which User or Device has been throttled.
+
+Learn more about [Ingestion Debugger](https://help.amplitude.com/hc/en-us/articles/360044835531).
+#### User Lookup
+
+After you fired an event, you can navigate to the `User & Account Look-Up` tab in Amplitude validate your event.
+
+Learn more about [User Lookup](https://www.docs.developers.amplitude.com/data/debugger/#user-lookup).
+
+#### Event Explorer
+In any Amplitude chart, click `Select events using your product`. You are able to watch the events roll in, in near-real time.
+
+Learn more about [Event Explorer](https://help.amplitude.com/hc/en-us/articles/360050836071-Event-Explorer-View-event-streams-in-real-time).
+#### Instrument Explorer
+Download the Google Chrome Extention which helps you examine and debug your Amplitude Browser SDK.
+
+Learn more about [Instrument Explorer](https://chrome.google.com/webstore/detail/amplitude-event-explorer/acehfjhnmhbmgkedjmjlobpgdicnhkbp).
+
+## 4. Enforce event schemas (Ampli)
+The Ampli CLI, Ampli Wrapper, and Amplitude SDK work together to generate a tracking library based on your Tracking Plan.
```

Comment:
Can you please add an example code snippet here to show the benefits of Ampli.

```
ampli.eventType({ stringProp: "", booleanProp: true })
```

Reply:
@justin-fiedler  Do you think this example needs to include all the language?

Reply:
Yes, I think it would be good to have it match the pattern of the other snippets.